pkg/cmd/cli/cmd/newapp.go

```diff
@@ -166,6 +166,8 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 			}
 		}
 	}
+	fmt.Fprintln(c.Out(), "Run 'oc status' to view your app.")
```

Comment:
Updated `new-app` use it's misnamed `fullName` to do the substitution.  `project` is borked the other way around, so we don't have a partial name.